Code/Tools/FBuild/FBuildCore/Error.cpp

```diff
@@ -458,6 +458,13 @@
     FormatError( iter, 1060u, function, "Can't modify frozen variable '%s'", var->GetName().Get() );
 }
 
+// Error_1061_MissingOpenBracket
+//------------------------------------------------------------------------------
```

Comment:
It seems this change adds 3 new errors, but they don't seem to be documented.

Reply:
Oh, I have been never aware of the docs of errors.
Added the docs in 0c630f3904e0917682da21d00a3ddb43c5cf0aa6. 
And I found that one of the errors will never be raised, and it was removed in the same commit.

BTW, I found that there are a few docs (e.g. 1042.html) are outdated, and a few errors (e.g. 1254) are no longer used.
I would like to clean them when I get time.

Reply:
I am going to click the "reslove" button for I didn't receive further feedback.
* docs for 2 errors were added.
* 1 error was removed.